### PR TITLE
Update for Silverblue 31

### PR DESCRIPTION
**Pull request by returntrip**

### CODE REVIEW

**Comment by returntrip** [comment]
Closes https://github.com/fedora-silverblue/silverblue-site/issues/74

File: pages/download/index.html [hunk 1]
```diff
@@ -24,26 +24,26 @@ <h1>Download a Silverblue image</h1>
...
 this checksum file
 </a> by following
-<a href="https://docs.fedoraproject.org/en-US/fedora/f30/install-guide/install/Preparing_for_Installation/#sect-verifying-images">
+<a href="https://docs.fedoraproject.org/en-US/fedora/f31/install-guide/install/Preparing_for_Installation/#sect-verifying-images">
 these instructions
 <a>.
 </p>
 
-<h1>Try Fedora 31 Beta</h1>
+<h1>Try Fedora 32 Beta</h1>
```
Comment (miabbott) [inline pages/download/index.html]: If we don't have F32 branded ISOs yet, let's just hide this whole section for now.

**Comment by miabbott** [comment]
One suggestion, but otherwise LGTM

File: pages/download/index.html (hunk 1 above)
Comment (returntrip) [reply pages/download/index.html]: Check the download link added "Coming Soon" there

Comment (dustymabe) [reply pages/download/index.html]: SGTM, I don't know if that's easy to "hide" or if we just need to delete the section.

Comment (returntrip) [reply pages/download/index.html]: image

Comment (dustymabe) [reply pages/download/index.html]: @returntrip could we just comment out the whole section?

**Comment by returntrip** [comment]
Please check now

**Pull request merged**